### PR TITLE
CB-17956 Remove Datahub validation from Datalake upgrade preparation

### PR DESCRIPTION
**Pull request by lacikaaa**
See detailed description in the commit message.

### CODE REVIEW

**Review by keyki — APPROVED** [review]
Tested locally.

File: datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java [hunk 1]
```diff
@@ -80,13 +82,13 @@ public class SdxRuntimeUpgradeService {
     @Inject
     private SdxUpgradeFilter upgradeFilter;
 
-    public SdxUpgradeResponse checkForUpgradeByName(String clusterName, SdxUpgradeRequest upgradeSdxClusterRequest, String accountId) {
```
Comment (lturcsanyi) [inline datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java:83]: Is there any reason why its called `prepare` here and `upgradePreparation` elsewhere?

**Review by lturcsanyi — APPROVED** [review]

File: datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java (hunk 1 above)
Comment (pdarvasi) [reply datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java:83]: +1 on this

**Review by pdarvasi — APPROVED** [review]

File: datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java (hunk 1 above)
Comment (lacikaaa) [reply datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java:83]: if it bothers you I can rename

**Pull request merged**